### PR TITLE
Nanodot 269e

### PR DESCRIPTION
**Pull request by Nanodot**
tried a second file - no write permission on google sheets yet so still shows unclaimed.

### CODE REVIEW

File: masks2/00269_e_083fdc4bd2407595_2021-10-10--09-55-29_3_886.png
Comment (Nanodot) [inline masks2/00269_e_083fdc4bd2407595_2021-10-10--09-55-29_3_886.png:1]: wrong file -trying to delete this mistaken file location

**Comment by doktorsleepelss** [comment]
00268_e is missing the lane lines.  And you have a gap between the road and mycar.  Have to overlap.

**Comment by Nanodot** [comment]
[image: 00268_e_6bb51d17e6d977e4_2021-09-21--20-42-16_2_898 (1).png]
I noticed those things after i PR'd the file. The gap was my mistake as
part of my learning curve of the application. The loss of lane lines was
due to that layer being shifted back later work. I uploaded this (attached)
file to github in an attempt to replace the file. Somehow i think it ended
up in my PR for 269e by mistake. The image we are talking about is 268e as
i screwed up the PR process.

This whole forking, branches, and pushing is a part of github i have never
done. I understand the concepts/theory of each thing, but the software
interface seems very awkward to me. Thanks for emailing!!!

**Comment by doktorsleepelss** [comment]
The checks issue has been fixed.  You need to make a new PR though.

**Pull request closed without merging**